packages/nocms-plugin-image-resources/test/createImageResourceProvider.spec.js

```diff
@@ -0,0 +1,94 @@
+import assert from 'assert';
+// import { createImageResourceProvider } from '../lib/createImageResourceProvider.js';
+import { activate } from '../lib/index.js';
+import fs from 'fs';
+
+const filePaths = [
+	'test/test-images/50mm-249.jpg',
+	'test/test-images/8-tallet.jpg'
+];
+
+let resourceProviders = [];
+let pluginActivationContext;
+const fakeFiles = [];
+const writtenFiles = [];
+
+function fileToBuffer(path){
+	return new Promise((resolve, reject) => {
```

Comment:
You could just do

```
const { promisify } = require('util');
const readFileAsync = promisify(fs.readFile);
```
or use experimental fs promises:
```
await fs.promises.readFile(...);
```
https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_promises_api